### ISSUE TITLE
OS autodetection

### ISSUE THREAD

**Issue by puerco** [chainguard-dev/darkfiles#7]
We should enable `darkfiles` to autodetect what kind of OS it is looking at inside the image.

**Comment by SetsuY** [comment]
When I did DiffBOM I just look at known directories of package managers and see if scanning returns an SBOM. Perhaps that's a quick and easy way?

**Comment by SetsuY** [comment]
#8 Did some light testing and it works

**Comment by puerco** [comment]
Yes exactly just adding some logic that looks for the expected files of each package manager should do trick. :+1:
